core/server/web/api/v2/admin/app.js

```diff
@@ -27,7 +27,6 @@ module.exports = function setupApiApp() {
 
     // Admin API shouldn't be cached
     apiApp.use(shared.middlewares.cacheControl('private'));
-
```

Comment:
This is probably a miss-commited change :thinking: No need to change formatting for this file